MEMBER_TRAVEL_FUND.md

```diff
@@ -10,7 +10,8 @@ and the Node.js Foundation projects.
 
 ## Process
 
-Any member can apply for travel funds.
+Any member can apply for travel funds. Requests must be made prior to the start
```

Comment:
Do the requests need to be approved prior to the start of travel? We should probably clarify that.

Reply:
I was thinking "no" as it's not the submitter's fault if TSC or CommComm takes a week to approve something instead of 72 hours or whatever.

I'm thinking let's go with the smallest possible restriction/modification to the policy and we can always go further later.

Reply:
I updated the text to hopefully clarify the point.

Reply:
Hmmm... we should give a reasonable time frame before here, then... because the way this is written, I can submit the request 5 minutes before my flight leaves. Perhaps...

`Request must be made 72 hours prior to the start of travel`.

Reply:
Actually I'm going to undo that change since I think it opens up more questions than it answers, at least the way I worded it. Shouldn't rush these things....

Reply:
> I can submit the request 5 minutes before my flight leaves

Yeah, that was the point. That should be OK in my opinion. You've already spent the money on airfare, and you're already going to be paying for lodging out of pocket. Why should 72 hours be required?

Reply:
I'm +1 as is. Any cutoff is arbitrary and I can't think why some time earlier than the start of travel is better.

Reply:
The new second paragraph addresses my original concern. I'm good with this.